### PR TITLE
Add release for Azure v14.1.7

### PR DESCRIPTION
**Pull request by tuommaki**
Towards https://github.com/giantswarm/giantswarm/issues/17740

### CODE REVIEW

**Comment by tuommaki** [comment]
```
Error from server (BadRequest): admission webhook "validate.cluster.update.azure-admission-controller-unique.giantswarm.io" denied the request: downgrading is not allowed error: downgrading is not allowed (attempted to downgrade from 15.0.0 to 14.1.7-1624949409)
```

Ok. The test didn't work as expected :slightly_frowning_face:

**Comment by whites11** [comment]
> ```
> Error from server (BadRequest): admission webhook "validate.cluster.update.azure-admission-controller-unique.giantswarm.io" denied the request: downgrading is not allowed error: downgrading is not allowed (attempted to downgrade from 15.0.0 to 14.1.7-1624949409)
> ```
> 
> Ok. The test didn't work as expected

yes upgrade test only works with latest release

**Comment by tuommaki** [comment]
Regarding upgrades:

I tested following upgrades yesterday:
- `13.1.1` -> `14.0.2` -> `14.1.7`
- `14.1.6` -> `14.1.7`

All upgrades were successful in the sense that the process progressed normally, nodes got replaced and status conditions resolved as expected. I think the average time was slightly under 30min for default cluster (1 master, 3 workers in one node pool).

File: azure/v14.1.7/announcement.md [hunk 1]
```diff
@@ -0,0 +1 @@
+**Workload cluster release v14.1.7 for Azure** is now available. This release upgrades Kubernetes to v1.19.12.
```
Comment (tuommaki) [inline azure/v14.1.7/announcement.md:1]: Ping @T-Kukawka - I'm not sure if there's anything more to say. AFAICS there's no major fixes between 1.19.10 - 1.19.12. Bug fixes for Azure, yes, but no CVEs that impact us nor any other critical changes. Is that enough for announcement?

Comment (whites11) [reply azure/v14.1.7/announcement.md:1]: makes sense for me

File: azure/v14.1.7/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# :zap: Giant Swarm Release v14.1.7 for Azure :zap:
+
+<< Add description here >>
+
+## Change details
+
+
+### azure-operator [5.5.4](https://github.com/giantswarm/azure-operator/releases/tag/v5.5.4)
+
+Not found
```
Comment (whites11) [inline azure/v14.1.7/README.md:10]: We have to change this with something else.

Comment (tuommaki) [reply azure/v14.1.7/README.md:10]:
```suggestion
Meta release for guaranteed node rolling. No changes in functionality.
```
Something like this or should I drop it altogether?

Comment (whites11) [reply azure/v14.1.7/README.md:10]: No better leave it, LGTM

File: azure/v14.1.7/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# :zap: Giant Swarm Release v14.1.7 for Azure :zap:
+
+<< Add description here >>
```
Comment (whites11) [inline azure/v14.1.7/README.md:3]: OK last change here (I guess something very similar to the announcement contents will do) and then we're good to go

File: azure/v14.1.7/announcement.md (hunk 1 above)
Comment (T-Kukawka) [reply azure/v14.1.7/announcement.md:1]: yeah sounds good 👍

File: azure/v14.1.7/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
```
[the remark this answers is not in the document]
Comment (whites11) [reply azure/v14.1.7/README.md:3]: LGTM

**Review by whites11 — APPROVED** [review]

**Pull request merged**